packages/hydrogen/src/storefront.ts

```diff
@@ -116,9 +118,12 @@ function minifyQuery(string: string) {
     .trim();
 }
 
+const SHOPIFY_Y = '_shopify_y';
+const SHOPIFY_S = '_shopify_s';
```

Comment:
@frehner We need to export those cart constants

Reply:
From hydrogen-react? Yeah I had that exact same thought when I saw that haha